.eslintrc.js

```diff
@@ -786,19 +786,6 @@ module.exports = {
         ],
         'react/jsx-uses-vars': 'error',
         'react/jsx-uses-react': 'error',
-        'import/no-restricted-paths': [
-            'error',
-            {
-                basePath: './src',
-                zones: [
-                    {
```

Comment:
We still want this don't we? Maybe just changed?

Reply:
I'll be submitting a separate PR for that, easier to revert than selectively changing

Reply:
Done